units/systemd-journald.service.in

```diff
@@ -29,6 +29,7 @@ IgnoreOnIsolate=yes
 DeviceAllow=char-* rw
 ExecStart={{LIBEXECDIR}}/systemd-journald
 FileDescriptorStoreMax=4224
+ImportCredential=journald.*
```

Comment:
hmm, so far none of our system credentials carry the "d" suffix. maybe drop it here too, i.e. do `journal.*` here, and rename the cred you care about?